server/src/main/java/org/elasticsearch/index/shard/IndexingStats.java

```diff
@@ -92,6 +99,7 @@ public void add(Stats stats) {
             if (isThrottled != stats.isThrottled) {
                 isThrottled = true; // When combining if one is throttled set result to throttled.
             }
+            writeLoadAvg = ((writeLoadAvg + stats.writeLoadAvg) / 2);
```

Comment:
I think we should leave out `/ 2`? As an example, if we have one shard that has a load of 4 cpus and another shard that has a load of 2 cpus, we have a total load of 6 cpus, not 3?

Reply:
It all depends wether we want to aggregate the total load per index or compute the average load across all shards. I think it makes sense to get an aggregate view at the index level instead of the average load. I'll change it.

Reply:
:+1: also it aggregates to the node level too, where it would be sort of strange not to have the summed up load?